app/Http/Resources/RequestNcpoListingResource.php

```
@@ -0,0 +1,26 @@
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class RequestNcpoListingResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @return array<string, mixed>
+     */
+    public function toArray(Request $request): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)